sparse/etc/acdbdata

```diff
@@ -0,0 +1 @@
+/system/etc/acdbdata
```

Comment:
Could you please rename this commit message instead saying why acdbdata symlink is needed?